wikipron/config.py

```diff
@@ -54,12 +55,13 @@ def __init__(
         phonetic: bool = False,
         dialect: Optional[str] = None,
         require_dialect_label: bool = False,
+        segment: bool = True,
```

Comment:
+1

Reply:
Great! This sounds good, I'll switch this over too.

Reply:
So it's still failing the tests because it defaults to segmentation, and the `expected_pron` variables are not equipped for this. 
Should I combine the tests, or should I just add spaces to the `expected_pron`'s?